src/component/parser/json/Definitions.ts

```diff
@@ -4,17 +4,22 @@ import SequenceFlow from '../../../model/bpmn/edge/SequenceFlow';
 import BpmnModel from '../../../model/bpmn/BpmnModel';
 import DiagramConverter from './converter/DiagramConverter';
 import SemanticConverter from './converter/SemanticConverter';
+import CollaborationConverter from './converter/CollaborationConverter';
 
 @JsonObject('definitions')
 export class Definitions {
+  @JsonProperty('collaboration', CollaborationConverter)
+  private readonly _collaboration: Collaboration;
+
   @JsonProperty('process', SemanticConverter)
   private readonly _semantic: Semantic;
```

Comment:
I'll do a PR to rename Semantic in Process ;)

Reply:
see #121